src/Discovery/DiscoveryDetails/Components/FieldGrouping/TabField/DataDownloadList/DataDownloadList.tsx

```diff
@@ -46,12 +46,6 @@ const DataDownloadList = ({
   if (discoveryConfig.features?.authorization?.enabled) {
     userHasAccessToDownload = resourceInfo[accessibleFieldName] === AccessLevel.ACCESSIBLE;
   }
-  // disable user's access if there's no manifest found for this study
-  const exportToWorkspaceConfig = discoveryConfig.features.exportToWorkspace;
-  const { manifestFieldName } = exportToWorkspaceConfig;
-  if (!resourceInfo[manifestFieldName] || resourceInfo[manifestFieldName].length === 0) {
-    userHasAccessToDownload = false;
-  }
```

Comment:
This is removed because we will rely on using `isManifestDataMissing()` insead of this